server/src/modules-resolver.ts

```diff
@@ -12,3 +14,20 @@ export function findPathToModule(dir: string, moduleName: string): string|undefi
         return undefined;
     }
 }
+
+export function findPathToSdk(
+    dir: string,
+    moduleName: string
+): string | undefined {
+    const stat = fs.statSync(dir);
+    if (stat.isDirectory()) {
+        const candidate = path.resolve(dir, '.yarn/sdks', moduleName);
```

Comment:
Please separate two directories so that those are joined with proper separator:
```suggestion
        const candidate = path.resolve(dir, '.yarn', 'sdks', moduleName);
```